document/CoreFormat.mdk

```diff
@@ -303,51 +316,116 @@ an analysis run. The value of `invocationInfo` would have to be combined with in
 source to allow the run to be repeated.
 ~
 
-### `analysisTargets` property { #runInfo-analysisTargets }
+### `fileInfo` property { #runInfo-fileInfo }
 
-A `runInfo` object may contain a property named `analysisTargets` whose value is an array,
-each of whose elements is a `fileReference` object (&sect;[#fileReference])
-representing the location of a single analysis target scanned during the run.
-This array shall contain one entry for each analysis target that was scanned,
-even if the analysis targets were not individually specified on the command line.
+A `runInfo` object should contain a property named `fileInfo` whose value is a JSON object,
+each of whose properties represents a file that was scanned in the course of the run.
 
-~ IsoNumberedNote { @IsoNumberedNote=1 }
-The command line with which the tool was invoked might specify its input files
-by means of a wild card such as *.cc, or it might specify them implicitly,
-for example, by scanning the files in the current directory.
+The `fileInfo` object should contain properties representing at least those files in which results were detected,
+but it may contain properties representing all scanned files (whether or not results were detected in those files),
+or for any subset of those files.
+
+~ IsoNumberedNote { @IsoNumberedNote=1}
+`fileReference` objects contain information that is useful for viewers.
+Viewers will be able to provide the most information to users if the `fileInfo`
+property is present and contains information for every file in which results were detected.
 ~
 
-The `analysisTargets` array shall be empty if no analysis targets were scanned in the course of the run.
+Each property name in the `fileInfo` object shall be the URI of a scanned file.
+No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
 
-~ IsoNumberedNote
-This could happen if the command line specified a wildcard such as *.cc for the input files,
-and no files matched the wildcard.
-~
+Each property value in the `fileInfo` object shall be an array of `fileReference` objects (&sect;[#fileReference])
+which contain information about the file identified by the URI in the property name.
 
-~ IsoExample
+If the scanned file is not nested within another file, then the array shall consist of a single
+element, and the property name shall be the absolute URI of the scanned file.
+
+~ IsoNumberedExample { @IsoNumberedExample=1}
 
 ```
-[
-    {
-        "uri": "file:///C:/Code/a.cc",
-        "hashes": [
+runInfo: {
+    fileInfo: {
+        "file:///C:/Code/main.cc": [
             {
-                "value": "b13ce2678a8807ba0765ab94a0ecd394f869bc81",
-                "algorithm": "SHA-256"
+                "uri": "file:///C:/Code/main.cc"
```

Comment:
I can certainly do this, but it's more than just a matter of eliminating this line from the sample. As the spec stands, `uri` is required.

I can change the spec to say that it's "required, except that if there's only one array element and uri is omitted, it's interpreted as being the same as the property name." ok?